src/components/CourseOutlineStatus/CourseOutlineStatus.scss

```diff
@@ -22,6 +22,6 @@
   margin-bottom: 0;
 }
 
-.text-info {
+.checklist-link {
```

Comment:
I left this in by accident, but this is why the link on the course outline page appeared a teal color, since `text-info` is a Bootstrap classname.

Reply:
By "by accident", I mean that this was meant for the PR that addressed the link color and alignment on the course outline page, which I was waking on before this.